polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -117,7 +117,10 @@
   #{$se23} & {
     border-bottom-left-radius: var(--p-border-radius-3);
     border-bottom-right-radius: var(--p-border-radius-3);
-    padding-bottom: var(--p-space-4);
+
+    &:not(.Section-flush) {
+      padding-bottom: var(--p-space-4);
+    }
```

Comment:
Looks like there _could_ be some conflict with the [LegacyCard Experimental updates here](https://github.com/Shopify/polaris/pull/9558/files#diff-7f3b0e011920dd647877d678aab9a8d3f550013c626dd1b6d8be21dc8d23ca51R124-R127) or around these styles.

cc @qt314 could you check this doesn't affect your current PR?